router/server.go

```diff
@@ -139,7 +139,7 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int, g http.
 	gMux.HandleFunc("/api/oam/{type}", h.OAMRegisterHandler).Methods("GET", "POST")
 	gMux.HandleFunc("/api/oam/{type}/{name}", h.OAMComponentDetailsHandler).Methods("GET")
 	gMux.HandleFunc("/api/oam/{type}/{name}/{id}", h.OAMComponentDetailByIDHandler).Methods("GET")
-	gMux.HandleFunc("/api/experimental/oam/{type}", h.OAMRegisterHandler).Methods("GET", "POST")
+	gMux.HandleFunc("/api/oam/{type}", h.OAMRegisterHandler).Methods("GET", "POST")
```

Comment:
```suggestion
	
```

We don't need to rename it. We can rather remove it. This is because we already have the updated API **3 lines up** 😅

Moreover, check this line: https://github.com/meshery/meshery/blob/6423b815ac7f610c9935f102d12e6bb428001544/router/server.go#L139